src-ts/tools/work/work-detail-details/work-detail-details-pane/WorkDetailDetailsPane.tsx

```diff
@@ -1,115 +1,64 @@
+import { Link } from '@reach/router'
 import _ from 'lodash'
-import { FC } from 'react'
+import { Dispatch, FC, SetStateAction, useEffect, useState } from 'react'
+
+import { LoadingSpinner } from '../../../../lib'
+import { workFactoryMapFormData } from '../../../../lib/work-provider/work-functions/work-factory'
 
 import styles from './WorkDetailDetailsPane.module.scss'
 
 interface WorkDetailDetailsPaneProps {
-    formData: any
-}
-
-const WorkDetailDetailsPane: FC<WorkDetailDetailsPaneProps> = ({ formData }: WorkDetailDetailsPaneProps) => {
-    const sections: Array<string> = ['basicInfo', 'websitePurpose', 'pageDetails', 'branding']
-
-    return (
-        <div className={styles['paneContent']}>
-            {sections
-                .filter((section) => {
-                    if (section === 'pageDetails') {
-                        return _.get(formData[section], 'pages[0].pageDetails') !== ''
-                    }
-                    return !!formData[section]
-                })
-                .map((section) => {
-                    return (
-                        <>
-                            {section === 'pageDetails'
-                                ? renderPageDetails(formData, section)
-                                : renderDetails(formData, section)
-                            }
-                        </>
-                    )
-                })}
-        </div>
-    )
+    formData: any,
+    isReviewPage?: boolean,
+    redirectUrl?: string
 }
 
-function renderPageDetails(formData: any, section: string): Array<JSX.Element> {
-    const items: { pages: [] } = formData[section] || {}
-    const pages: [] = items?.pages || []
+const WorkDetailDetailsPane: FC<WorkDetailDetailsPaneProps> = ({ formData, isReviewPage = false, redirectUrl = '' }: WorkDetailDetailsPaneProps) => {
+    const [details, setDetails]: [any, Dispatch<SetStateAction<any>>] = useState<any>(undefined)
```

Comment:
i know that prettier insists that there is no blank line at the start of functions, which is why i don't use prettier. i find it difficult to read/parse the function signature and that first line. please insert a blank line between the signature and first line of  implementation for functions that are more than a few lines long.